www-src/app/replicator/replicator_migration.coffee

```diff
@@ -0,0 +1,67 @@
+log = require('/lib/persistent_log')
+    prefix: "replicator_migration_sqlite"
+    date: true
+
+DBNAME = "cozy-files.db"
+DBPHOTOS = "cozy-photos.db"
+
+module.exports =
+    sqliteDB: null
+    sqliteDBPhotos: null
+
+    migrateDBs: (callback) ->
+        # Check db new db already good.
+        @getConfig @db, (err, hasConfig) =>
+            return callback err if err
+            if hasConfig
+                return callback null, 'db already configured'
+
+            # else check old db present.
+            @initSQLiteDBs()
+            @getConfig @sqliteDB, (err, hasConfig) =>
+                return callback err if err
+                unless hasConfig
+                    return callback null, 'nothing to migrate'
+
+                # else Migrate.
+                log.info 'Migrate sqlite db to idb'
+                @displayMessage() # Add message to the user
+                @replicateDBs (err) =>
+                    return callback err if err
+                    @destroySQLiteDBs callback
+
+
+    # Init old sqlite db
+    initSQLiteDBs: ->
+        @sqliteDBPhotos = new PouchDB DBPHOTOS, adapter: 'websql'
+        @sqliteDB = new PouchDB DBNAME, adapter: 'websql'
+
+    getConfig: (db, callback) ->
+        db.get 'localconfig', (err, config) =>
```

Comment:
Well, I was not clear. For websql, it's too late to use a local document but since nobody has a running idb PouchDB, it's probably a good time to make such a change.

For the why, I think it's better to save local configuration in a local document, and not just a normal pouchdb document to be sure that this configuration will never be replicated to another couchdb/pouchdb.